src/lib/geo-location.ts

```diff
@@ -8,21 +8,27 @@ const CACHE_TTL = 8.64e7
 // 10 seconds
 const REQUEST_TIMEOUT = 1e4
 
-/**
- * @typedef GeoLocation
- * @type {object}
- * @property {string} city
- * @property {object} country
- * @property {string} country.code
- * @property {string} country.name
- * @property {object} subdivision
- * @property {string} subdivision.code
- * @property {string} subdivision.name
- * @property {number} longitude
- * @property {number} latitude
- * @property {string} timezone
- */
-export const mockLocation = {
+export type Geolocation = {
+  city: string
+  country: {
+    code: string
+    name: string
+  }
+  subdivision: {
+    code: string
+    name: string
+  }
+  longitude: number
+  latitude: number
+  timezone: string
+}
+
+interface State {
+  get(key: string): unknown
+  set(key: string, value: unknown): void
+}
```

Comment:
I believe this is `CLIState`?
```suggestion
```

Reply:
Making this change would require either making lots of updates to the test suite or littering it with suppressions. This requires none, and only drags in the minimal portion of the state interface that this module relies on.